### ISSUE TITLE
Список зависимостей

### ISSUE THREAD

**Issue by ssten** [DikBSD/freeLib#8]
Я правильно понимаю, что:

> сore;xml;sql;network;gui;xmlpatterns;widgets;qml;positioning;printsupport;webchannel;quick;concurrent;webengine;webenginewidgets

это список используемых модулей (по которому строится список зависимостей)?

Можно это как-то отразить в README?

**Comment by DikBSD** [comment]
Да, все так. Я внесу эти зависимости в readme.

**Comment by ssten** [comment]
Пошарился по [списку](https://packages.gentoo.org/categories/dev-qt).
Получилось собрать следующее:

> dev-qt/qtcore
> dev-qt/qtxml
> dev-qt/qtsql
> dev-qt/qtnetwork
> dev-qt/qtgui
> dev-qt/qtxmlpatterns
> dev-qt/qtwidgets
> *qml*
>> qtdeclarative 	The QML and Quick modules for the Qt5 framework

> dev-qt/qtpositioning
> dev-qt/qtprintsupport
> dev-qt/qtwebchannel
> *quick*
>> qtquickcontrols 	Set of Qt Quick controls to create complete user interfaces (deprecated)
>> qtquickcontrols2 	Set of next generation Qt Quick controls for the Qt5 framework

> dev-qt/qtconcurrent
> dev-qt/qtwebengine
> *webenginewidgets*
>> designer 	WYSIWYG tool for designing and building graphical user interfaces with QtWidgets

Для трёх модулей однозначного соответствия навскидку установить не получилось…
Прошу проверить предположения.

**Comment by ssten** [comment]
Также в списке следует посчитать dev-libs/quazip (прошу переместить в каталог **3rdparty**, но это потом, когда я опишу опцию для использования системной версии).

**Comment by ssten** [comment]
Глупый вопрос: можно ли сделать зависимость от *dev-qt/qtwebengine* опциональной?
Ибо он досейчас тянет второго питона.

**Comment by DikBSD** [comment]
В предыдущей правке кода я исключил лишние зависимости. Теперь там нет даже `qtwebengine`  (см. файл src.pro)
Необходимые зависимости следующие: `sql xml network widgets xmlpatterns concurrent printsupport gui`

Я удалил `qtwebengine5-dev` , поскольку ограничил некоторые неиспользуемые возможности freeLib, которые зависели от qtwebengine5: 
```
sudo apt remove qtwebengine5-dev
```
Сборка прошла успешно.

Надо будет поправить README.md в строке установки нужных библиотек.

**Comment by ssten** [comment]
Это по троллям.
Спасибо.

**dev-libs/quazip** остаётся?

**Comment by DikBSD** [comment]
Да, пока остаётся. Чтобы его убрать, нужно огромную работу проделать. Можно будет заняться, когда основной функционал freeLib будет сделан.
